dev/com.ibm.ws.transaction.core_fat/test-applications/transaction/src/com/ibm/ws/transaction/web/SimpleServlet.java

```diff
@@ -541,11 +527,8 @@ public void testNEW2(HttpServletRequest request, HttpServletResponse response) t
      * LTC transaction.
      */
     @Test
-    // TODO: Remove skip when jdbc feature supports jakartaee9
-    @SkipForRepeat({ SkipForRepeat.EE9_FEATURES })
     public void testExplicitLTCCommit(HttpServletRequest request, HttpServletResponse response) throws Exception {
         InitialContext context = new InitialContext();
-        DataSource ds = (DataSource) context.lookup("java:comp/env/jdbc/derby");
         Connection con = ds.getConnection();
```

Comment:
We need to keep the DataSource field with the jndi lookup in the test methods.  Please reinstate the line of code in each method and comment it out, like so `//DataSource ds = (DataSource) context.lookup("java:comp/env/jdbc/derby");` This will return the test methods to their original content.

Reply:
done. thanks!